.scannerwork/report-task.txt

```diff
@@ -0,0 +1,6 @@
+projectKey=Sonar-weatherapp-backend
```

Comment:
what is that? do we need to commit it to github?

Reply:
the .scannerwork folder was automatically generated when I run Sonar Code Analysis locally on my computer, but makes sense do not commit it to github. I will add it to the gitignore file.